### DISCUSSION TITLE
srsRAN Project gNB with Quectel CC660D-LS

### DISCUSSION THREAD

**Discussion by mirunalinte** [srsran/srsRAN_Project#704] [General Help]
Hello everyone! Has anyone tried to configure and run a 5G NR SA NTN using the srsRAN Project gNB and the Quectel CC660D-LS radio module as a user equipment? Any advice would be appreciated! Thanks and best wishes.

**Comment by pgawlowicz** [comment]
Hi, it seems the Quectel CC660D-LS radio module is for NB-IoT NTN. srsRAN Project does not support it.
